fog/sql_recovery_db/src/lib.rs

```diff
@@ -53,6 +57,34 @@ pub const SQL_MAX_PARAMS: usize = 65000;
 /// Maximal number of rows to insert in one batch.
 pub const SQL_MAX_ROWS: usize = 5000;
 
+/// SQL recovery DB connection configuration parameters
+#[derive(Debug, Clone, StructOpt, Serialize)]
+pub struct SqlRecoveryDbParams {
+    /// idle timeouts
+    #[structopt(long, env, default_value = "60", parse(try_from_str=parse_duration_in_seconds))]
+    postgres_idle_timeout: Duration,
+    /// max lifetime
```

Comment:
Nit: Can you please put new lines between fields, and capitalize the first letter? I believe that has been the convention in other places.